examples/timeseries/timeseries_classification_from_scratch.py

```diff
@@ -0,0 +1,247 @@
+"""
+Title: Time series classification from scratch
+Author: [hfawaz](https://github.com/hfawaz/)
+Date created: 2020/07/21
+Last modified: 2020/08/21
+Description: Training a time series classifier from scratch on the FordA dataset from
+the UCR/UEA archive.
+"""
+"""
+## Introduction
+
+This example shows how to do time series classification from scratch, starting from raw
+CSV time series files on disk. We demonstrate the workflow on the FordA dataset from the 
+[UCR/UEA archive](https://www.cs.ucr.edu/%7Eeamonn/time_series_data_2018/).
+
+"""
+
+"""
+## Setup
+
+"""
+
+import tensorflow as tf
+from tensorflow import keras
+import numpy as np
+import matplotlib.pyplot as plt
+from sklearn.metrics import accuracy_score
+
+"""
+## Load the data: the FordA dataset
+
+### Description
+The dataset we are using here is called FordA.
+The data comes from the UCR archive.
+The dataset contains 3601 training instances and another 1320 testing instances.
+Each time series corresponds to a measurement of engine noise captured by a motor sensor.
+For this task, the goal is to automatically determine whether a certain symptom
+exists or not in an automotive system. The problem is a balanced binary classification
+task. The full description of
+this dataset can be found [here](http://www.j-wichard.de/publications/FordPaper.pdf).
+
+### Read the TSV data online
+We will use the `FordA_TRAIN` file for training and the
+`FordA_TEST` file for testing. The simplicity of this dataset
+allows us to demonstrate effectively how to use ConvNets for time series classification.
+In this file, the first column corresponds to the label.
+"""
+
+
+def readucr(filename):
+    data = np.loadtxt(filename, delimiter="\t")
+    y = data[:, 0]
+    x = data[:, 1:]
+    return x, y.astype(int)
+
+
+root_url = "https://raw.githubusercontent.com/hfawaz/cd-diagram/master/FordA/"
```

Comment:
The compressed archive contains 128 datasets. 
This tutorial is for one of the 128 datasets called FordA, so in order to avoid downloading the whole archive, I uploaded FordA on GitHub to use it in this tutorial. 
The archive is not that large (260 MB), so do you think it is better to download and unzip the archive online in the notebook ? 
Note: the authors of the archive require people searching for a password in a paper to unlock the archive.

Reply:
Ok, sounds good, let's keep your GitHub link then!